modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -611,6 +611,18 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         virtual ~Importer();
     };
 
+    /** @brief Creates the importer of <a href="https://pjreddie.com/darknet/">Darknet</a> framework network.
+    *  @param cfgFile      path to the .cfg file with text description of the network architecture.
+    *  @param darknetModel path to the .weights file with learned network.
+    *  @returns Pointer to the created importer, NULL in failure cases.
+    */
+    CV_EXPORTS_W Ptr<Importer> createDarknetImporter(const String &cfgFile, const String &darknetModel = String());
```

Comment:
We defined methods like `createCaffeImporter` as deprecated. Please keep only `readNetFromDarknet`.